packages/@ourworldindata/grapher/src/core/Grapher.tsx

```diff
@@ -926,6 +926,11 @@ export class Grapher
         return this.timelineHandleTimeBounds[1]
     }
 
+    @action.bound resetHandleTimeBounds(): void {
+        this.startHandleTimeBound = this.timelineMinTime || -Infinity
+        this.endHandleTimeBound = this.timelineMaxTime || Infinity
```

Comment:
```suggestion
        this.startHandleTimeBound = this.timelineMinTime ?? -Infinity
        this.endHandleTimeBound = this.timelineMaxTime ?? Infinity
```

Otherwise, we don't handle `0` correctly.
My general rule is to use `??` over `||` whenever possible, it's less surprising in general.